src/c2_blockchain/p2_extrinsic_state.rs

```diff
@@ -91,7 +91,7 @@ fn build_forked_chain() -> (Vec<Header>, Vec<Header>) {
     // There is a critical thinking question for you there.
 }
 
-// To run these tests: `cargo test part_1`
+// To run these tests: `cargo test part_2`
```

Comment:
Good catch on these being outdated. Ideally they would mirror the style of chapter 1. In chapter 1 which was about state machines each of them was `sm_1_`, `sm_2_`, etc.

So here in chapter 2 about blockchains, they should be `bc_1_`, `bc_2_`, etc. This can happen in another PR, and doesn't need to block this one. I just wanted to let you know my plan.

Reply:
Will fix this with this PR